### PR TITLE
[v6r20] Fix SE Cache

### PR DESCRIPTION
**Pull request by chaen**
@atsareg please continue using the release not script, it works super nicely ! :-)
This fixes https://github.com/DIRACGrid/DIRAC/issues/3727
Thanks @andresailer for testing it

BEGINRELEASENOTES

*DMS
FIX: use proxy location in the SECache

*RMS
FIX: use downloadVOMSProxyToFile in RequestTask

ENDRELEASENOTES

### CODE REVIEW

File: RequestManagementSystem/private/RequestTask.py [hunk 1]
```diff
@@ -158,13 +158,11 @@ def setupProxy( self ):
       return S_OK( { "Shifter": isShifter, "ProxyFile": proxyFile } )
 
     # # if we're here owner is not a shifter at all
-    ownerProxy = gProxyManager.downloadVOMSProxy( ownerDN, ownerGroup )
-    if not ownerProxy["OK"] or not ownerProxy["Value"]:
-      reason = ownerProxy["Message"] if "Message" in ownerProxy else "No valid proxy found in ProxyManager."
+    ownerProxyFile = gProxyManager.downloadVOMSProxyToFile( ownerDN, ownerGroup )
+    if not ownerProxyFile["OK"] or not ownerProxyFile["Value"]:
+      reason = ownerProxyFile["Message"] if "Message" in ownerProxyFile else "No valid proxy found in ProxyManager."
```
Comment (fstagni) [inline RequestManagementSystem/private/RequestTask.py]: `reason = ownerProxyFile.get('Message', "No valid proxy found in ProxyManager.")`

File: RequestManagementSystem/private/RequestTask.py [hunk 2]
```diff
@@ -158,13 +158,11 @@ def setupProxy( self ):
       return S_OK( { "Shifter": isShifter, "ProxyFile": proxyFile } )
 
     # # if we're here owner is not a shifter at all
-    ownerProxy = gProxyManager.downloadVOMSProxy( ownerDN, ownerGroup )
-    if not ownerProxy["OK"] or not ownerProxy["Value"]:
-      reason = ownerProxy["Message"] if "Message" in ownerProxy else "No valid proxy found in ProxyManager."
+    ownerProxyFile = gProxyManager.downloadVOMSProxyToFile( ownerDN, ownerGroup )
```
Comment (fstagni) [inline RequestManagementSystem/private/RequestTask.py]: pep8

**Comment by atsareg** [comment]
The nice GetReleaseNotes tool gives me now:

  subprocess.CalledProcessError: Command '['curl', '-s', '-H', 'Authorization: token xxxxxxxxxxxxxxxxxxxxxx', 'https://api.github.com/repos/DiracGrid/Dirac/pulls?state=closed&per_page=100']' returned non-zero exit status 4

It used to work for me but not now

**Comment by andresailer** [comment]
Release notes script is fixed in  #3735

**Pull request merged**